Public/Analog_clock.html

```diff
@@ -33,12 +33,40 @@
                     mn.style.transform = `rotateZ(${mm}deg)`;
                     sc.style.transform = `rotateZ(${ss}deg)`;
                 });
+
             </script>
-         <script data-name="BMC-Widget" data-cfasync="false" src="https://cdnjs.buymeacoffee.com/1.0.0/widget.prod.min.js" 
-                data-id="Astrodevil" data-description="Support me on Buy me a coffee!!" 
+         <script data-name="BMC-Widget" data-cfasync="false" src="https://cdnjs.buymeacoffee.com/1.0.0/widget.prod.min.js"
+                data-id="Astrodevil" data-description="Support me on Buy me a coffee!!"
                 data-message="" data-color="#40DCA5"
                 data-position="Right"
                 data-x_margin="50" data-y_margin="50">
-            </script>
+                </script>
+    <div class="mode">
+
+    <input type="checkbox" class="checkbox" id="checkbox">
+    <label for="checkbox" class="label">
+      <div class='ball'></div>
+    </label>
+    <script type="text/javascript">
+    const checkbox = document.getElementById("checkbox");
+  const body = document.querySelector("body");
+  const label = document.querySelector(".label");
+  const root = document.querySelector(":root");
+  checkbox.addEventListener("change", function() {
+    if (this.checked) {
+      body.style.background = "black";
+      label.style.backgroundColor = "#40DCA5";
+      root.style.setProperty('--sec-color', '#fff');
+    } else {
+      body.style.background = "skyblue";
+      label.style.backgroundColor = "white";
+      root.style.setProperty('--sec-color', 'red');
+
+    }
+  });
+</script>
```

Comment:
It is recommended to remove the JavaScript code from the HTML file since it doesn't make much sense to have it there when there is already an external JavaScript file attached. By separating the JavaScript code into its own file, it enhances code organization and maintainability.